extensions/ql-vscode/src/helpers.ts

```diff
@@ -590,5 +590,5 @@ export async function* walkDirectory(dir: string): AsyncIterableIterator<string>
  * Example: Returns "N repository" if N is one, "N repositories" otherwise.
  */
 export function pluralize(numItems: number | undefined, singular: string, plural: string): string {
```

Comment:
This function could do with some unit tests. They'd nicely document the expected behaviour too.

Also I if `numItems` is 0, then this will return e.g. `0 repository`. Is this definitely what we want (sounds a bit off grammatically to me)? Previously we'd return an empty string.

Reply:
It should return `0 repositories`, which sounds okay to me? 🤔  But you're right that it could do with some tests! Incoming...

Reply:
We deliberately changed it from the empty string because it resulted in a label like `0/` instead of `0/0 repositories`. I agree with Shati, the implementation looks correct now. But as said, tests will make this more clear.